src/pages/overview/pdf-electronic-seal-api/quickstarts/java/index.md

```diff
@@ -54,16 +54,17 @@ To complete this guide, you will need:
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.adobe.documentservices</groupId>
-  <artifactId>pdfservices-sdk-electronicseal-guide</artifactId>
-  <version>1</version>
+  <artifactId>pdfservices-sdk-samples</artifactId>
+  <version>${pdfservices.sdk.samples.version}</version>

```

Comment:
This line is not for the Samples version. 
This is a project in it's own regard. 
let us change it back to version 1

Reply:
Reverted.